src/cmd/cluster_intel_collector/config.go

```diff
@@ -0,0 +1,90 @@
+package cluster_intel_collector
+
+import (
+	"context"
+
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var (
+	log = newLogCollectorLogger("[log collector]")
+)
+
+type intelCollectorContext struct {
+	ctx           context.Context
+	clientSet     kubernetes.Interface // used to get access to logs
+	apiReader     client.Reader        // used for manifest collection
+	namespaceName string               // the default namespace ("dynatrace") or provided in the command line
+	toStdout      bool
+	targetDir     string
+}
+
+type manifestSpec struct {
```

Comment:
Agreed, was not happy with `manifestSpec` either. But I'll move the list of object queries into a separate file, cause it just makes the `manifests.go` file unnecessarily long otherwise.